### PR TITLE
Fix markdown format

### PR DESCRIPTION
**Pull request by pexcn**
Fix link.

### CODE REVIEW

File: languages/README_CHT.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ sudo /Applications/Install\ macOS\ Mojave.app/Contents/Resources/createinstallme
 ## Catalina
 sudo /Applications/Install\ macOS\ Catalina.app/Contents/Resources/createinstallmedia --volume /Volumes/MyVolumeName
 ```
-- 安裝 OpenCore 到你的 USB 上（若要 OpenCore release，請到: https://github.com/acidanthera/OpenCorePkg/releases）
+- 安裝 OpenCore 到你的 USB 上（若要 OpenCore release，請到: [https://github.com/acidanthera/OpenCorePkg/releases](https://github.com/acidanthera/OpenCorePkg/releases)）
```
Comment (dhinakg) [inline languages/README_CHT.md]: @doesprintfwork, can you please reword this so it's "to download the OpenCore release, [click here](https://github.com/acidanthera/OpenCorePkg/releases)"?

**Comment by doesprintfwork** [comment]
[按此](https://github.com/acidanthera/OpenCorePkg/releases)下載 OpenCore relase

File: languages/README_CHT.md (hunk 1 above)
Comment (doesprintfwork) [reply languages/README_CHT.md]: [按這裡](https://github.com/acidanthera/OpenCorePkg/releases)下載 OpenCore release

**Comment by pexcn** [comment]
Hi, now updated.

**Pull request merged**